### PR TITLE
Add Support for TopK Filter Commands

### LINKED ISSUE #11: Support for TopK Filter Commands
See [https://oss.redislabs.com/redisbloom/TopK_Commands/](https://oss.redislabs.com/redisbloom/TopK_Commands/)

### PR DESCRIPTION
**Pull request by dengliming**
fix #11

### CODE REVIEW

**Comment by filipecosta90** [comment]
@dengliming can we update from master? ( it will trigger CI )

**Comment by dengliming** [comment]
@dengliming sure.

**Comment by filipecosta90** [comment]
> @filipecosta90 sure.

Thank you =) I tried to push it to your master but was not able to do so ( on your other branch I was able )

**Comment by dengliming** [comment]
@filipecosta90 Wrong Operation. Sorry about that. I will submit a new PR from upstream. dimiss this one. : )

**Comment by dengliming** [comment]
@filipecosta90 Sorry for the wrong operation. Reopen now.

File: client_test.go [hunk 1]
```diff
@@ -107,3 +107,41 @@ func TestExists(t *testing.T) {
...
+	rets, err := client.TopkAdd(key, []string{"test"})
+	assert.Nil(t, err)
+	assert.NotNil(t, rets)
+	queryRet, err := client.TopkQuery(key, []string{"test", "nonexist"})
+	assert.Nil(t, err)
+	assert.Equal(t, 2, len(queryRet))
+	assert.Equal(t, int64(1), queryRet[0])
+	assert.Equal(t, int64(0), queryRet[1])
+
+	keys, err := client.TopkList(key)
```
Comment (ashtul) [inline client_test.go:144]: Can you add a test to check we are receiving/parsing correctly values in list?

**Pull request merged**